3 - PB/Documentazione esterna/Manuale Utente/Manuale Utente.typ

```diff
@@ -85,6 +86,39 @@ Per i requisiti di sistema e hardware si rimanda al documento #st_v.
 
 = Istruzioni d'uso
 
+== Creazione zona
+
+Successivamente alla creazione dell'ambiente di lavoro è possibile creare le zone contenenti i bin in modo personalizzato.
+
+Sulla sinistra della schermata è presente il pannello dedicato alla visualizzazione delle zone, dei prodotti, degli ordini e delle impostazioni.
+
+Se da esso si seleziona la voce "Zone" è possibile visualizzare la lista delle zone già presenti, elencate in un'apposita sezione nella quale è presente un pulsante nero contenente un "+".
+Alla sua pressione verrà reso disponibile, sulla destra della schermata, il pannello "Nuova zona" nel quale sarà possibile inserire tutti i dati utili alla creazione della zona personalizzata.
+
+I dati necessari alla creazione sono:
+- *id*: numero intero che rappresenta il codice identificativo univoco della zona;
+- *direzione*: può assumere valore "NS" o "EW" e rappresenta l'orientamento della zona rispetto al piano;
+- *dimensioni*:
+  - *profondità*: numero reale che definisce la profondità della zona;
+  - *larghezza*: numero reale che definisce la larghezza della zona. Essa è definibile solo se si procede con la configurazione delle colonne in modalità "colonne uguali", altrimenti nello stesso campo viene mostrata automaticamente la larghezza calcolata in base alle larghezze delle singole colonne;
+  - *altezza*: numero reale che definisce l'altezza della zona. Essa viene mostrata automaticamente in funzione delle singole altezze dei livelli.
+
+- *selezione tipo colonne*:
+  tramite un pulsante di opzione, è possibile selezionare la modalità di configurazione delle colonne.
+  Esse sono:
+    - *colonne uguali*: tramite l'opzione "\# colonne uguali" è possibile dichiarare che tutte le colonne della zona hanno larghezza uguale e definire il *numero di colonne* con un numero intero. In questo modo la larghezza di ogni singola colonna sarà uguale alla divisione tra la larghezza della zona dichiarata e il numero di colonne;
+    - *colonne personalizzate*: tramite l'opzione "Colonne personalizzate" è possibile specificare la *larghezza delle colonne* singolarmente dentro ad un apposito form separando ciascun valore (rappresentato da un numero reale) con degli spazi.
+    Per esempio, se viene inserito:
```

Comment:
Andare a capo una volta in più prima di questa riga e aggiungere un tab a questa e alle due righe seguenti, in modo che siano indentate correttamente nel pdf